examples/pyplots/annotation_basic.py

```diff
@@ -17,6 +19,4 @@
 ax.annotate('local max', xy=(2, 1), xytext=(3, 1.5),
             arrowprops=dict(facecolor='black', shrink=0.05),
             )
-
-ax.set_ylim(-2,2)
-plt.show()
```

Comment:
Shouldn't all the examples have `plt.show()` at the end so they can be run as `python example.py`?

Reply:
Feel free to fix this.